examples/catalog/android/.gitignore

```diff
@@ -1,9 +1,5 @@
-*.iml
 .gradle
-/local.properties
-/.idea/workspace.xml
-/.idea/libraries
-.DS_Store
+local.properties
 /build
 /captures
 GeneratedPluginRegistrant.java
```

Comment:
Seems like the tool re-writes them each time and it's in the git ignore for creation templates. Should we just ignore all the .java/.h/.m in our repo too? cc @mit-mit

Reply:
Just the `GeneratedPluginRegistrant.java` is generated. Any other files would be source.